test/cql-pytest/test_keyspace.py

```diff
@@ -97,15 +97,22 @@ def test_drop_keyspace_nonexistent(cql):
         cql.execute('DROP KEYSPACE nonexistent_keyspace')
 
 # Test trying to ALTER a keyspace.
-def test_alter_keyspace(cql, this_dc):
+def test_alter_keyspace(cql, this_dc, has_tablets):
```

Comment:
let's rename the test case to `test_alter_keyspace_durable_writes` since it's specific to that option